CONTRIBUTING.md

```diff
@@ -245,3 +245,34 @@ npm run update_prebuilt
 
 That last step is easy to forget, so try to make it a habit whenever you're
 prepping a PR that includes frontend work.
+
+### MacOS
```

Comment:
I would move this whole thing up to around line 182, just under the Debian/Ubuntu install steps; it still belongs under the 'Setting Up Your Dev Environment' heading.